### DISCUSSION TITLE
npm audit vulnerability report

### DISCUSSION THREAD

**Discussion by koolamusic** [typegoose/typegoose#508] [Q&A] [answered]
running `npm audit` generates the following

# npm audit report

```
mquery  <3.2.3
Severity: moderate
Prototype Pollution - https://npmjs.com/advisories/1649
fix available via `npm audit fix --force`
Will install @typegoose/typegoose@7.4.2, which is a breaking change
node_modules/mquery
  mongoose  3.7.0 - 5.11.6
  Depends on vulnerable versions of mquery
  node_modules/mongoose
    @typegoose/typegoose  7.4.3 - 8.0.0-beta.1
    Depends on vulnerable versions of mongoose
    node_modules/@typegoose/typegoose

3 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

```

According to the docs there is a hard stand on `mongoose@5.10.18` 

### Currently installed
`mongoose@5.10.18` 
`@typegoose/typegoose": "^7.4.8",`

Nodejs: version 12 LTS

I would like to know how to fix this issue, and which versions of either `typegoose` or mongoose would make sense to use.

**Comment by hasezoey** [comment, marked as answer]
if you search for an explanation on why 5.10.18, look [at the FAQ](https://typegoose.github.io/typegoose/docs/guides/faq#why-is-74x-constrained-to-mongoose-51018)

mongoose 5.11.19+ is supported since version `8.0.0-beta.2`, its on npm, you can try it (yes, mongoose was an breaking change, that is why 8.0.0)

**Reply by koolamusic** [reply]
Going to give this a shot 🤞🏾
